sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2335,6 +2335,17 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val UI_EXPLAIN_MODE = buildConf("spark.sql.ui.explainMode")
+    .doc("Configures the query explain mode used in the Spark SQL UI. The value can be 'simple', " +
+      "'extended', 'codegen', 'cost', or 'formatted'. The default log level is 'formatted'.")
+    .version("3.1.0")
+    .stringConf
+    .transform(_.toUpperCase(Locale.ROOT))
+    .checkValue(mode => Set("SIMPLE", "EXTENDED", "CODEGEN", "COST", "FORMATTED").contains(mode),
+      "Invalid value for 'spark.sql.queryDescriptionModeInListeners'. Valid values are " +
```

Comment:
Please rename this together, @maropu .

Reply:
This seems to be a leftover after conf renaming. Maybe, `spark.sql.ui.explainMode`?